### PR TITLE
Add the cache size output column

### PR DESCRIPTION
**Pull request by movie0125**
目前只有缓存page数量，但是没有缓存大小，增加一列输出缓存大小

截图_选择区域_20210919150450
截图_选择区域_20210919150438
截图_选择区域_20210919150425
截图_选择区域_20210919150653

### CODE REVIEW

**Comment by movie0125** [comment]
@silenceshell 请帮忙审核代码，谢谢

**Comment by silenceshell** [comment]
Could we change cache size in MB or GB for different size? This will improve readability.

File: formats.go [hunk 1]
```diff
@@ -267,3 +270,27 @@ func (stats PcStatusList) maxNameLen() int {
 	}
 	return maxName
 }
+
+func Size_to_string(cachesize int64) string {
+	var KB, MB, GB, TB, PB int64
```
Comment (silenceshell) [inline formats.go:275]: those could be defined as const vars.

File: formats.go [hunk 2]
```diff
@@ -267,3 +270,27 @@ func (stats PcStatusList) maxNameLen() int {
...
+	KB = 1024
+	MB = 1024 * KB
+	GB = 1024 * MB
+	TB = 1024 * GB
+	PB = 1024 * TB
+
+	if cachesize > PB {
```
Comment (silenceshell) [inline formats.go:282]: we could use `switch... case` for different cases.

**Comment by ruishawn** [comment]
Hi @silenceshell,  I already comunicated with @movie0125  and have another commit [21](https://github.com/silenceshell/hcache/pull/21) . Please review  it, tks!

**Pull request closed without merging**